### PR TITLE
Updated limitations page to reflect beta

### PR DESCRIPTION
**Pull request by Joelkw**
Made the necessary updates! 

@coury-clark I would love your eyes mostly on the performance section – does the way I am explaining it make sense? Feel free to make suggestions there.

### CODE REVIEW

**Review by coury-clark — APPROVED** [review]
Looks good! My only thought is that you have a section at the bottom about pre-3.31, but most of this page also isn't relevant to pre-3.31 that isn't called out.

File: doc/code_insights/explanations/current_limitations_of_code_insights.md [hunk 1]
```diff
@@ -1,20 +1,48 @@
...
+To accurately return historical data for insights running over all of your repositories, the backend service must run a large number of Sourcegraph searches. This means that unlike code insights running over just a few repositories, results are not returned instantly, but more often on the scale of 20-120 minutes, depending on:
 
-We're currently developing a scalable backend service that fixes this limitation, with a planned release by September 2021, that will allow you to run code insights over thousands of repositories at once. As of now, performance generally gets noticeably slower around 50 repositories, and becomes functionally unusable above 200 repositories.
+* _N_: how many repositories you have connected to your instance; in our tests, we used 26,400 repositories
+* _q_: the performance and resources of your Sourcegraph code insights instance in queries-per-second; in our tests, 7 queries per second was average
+* _c_: how well we can "compress" repositories so we don't need to re-run queries every month (e.g., if a respoitory hasn't changed in two months); in our tests, C = ~4
```
Comment (coury-clark) [inline doc/code_insights/explanations/current_limitations_of_code_insights.md:13]: Cool! I like that this using the generic `c` removes the need to be concerned with interval count, but can still give a useful answer.

Comment (Joelkw) [reply doc/code_insights/explanations/current_limitations_of_code_insights.md:13]: ah nice point, didn't even notice that – thanks!

**Comment by Joelkw** [comment]
> Looks good! My only thought is that you have a section at the bottom about pre-3.31, but most of this page also isn't relevant to pre-3.31 that isn't called out.

@coury-clark good point – I made it clearer that this page is current with an italicized sentence at the top, and that all "older" limitations are collected at the bottom.

**Pull request merged**